examples/aqlm_example.py

```diff
@@ -0,0 +1,42 @@
+from vllm import LLM, SamplingParams
+import argparse
+
+
+def main():
+
+    parser = argparse.ArgumentParser(
+        description='Example script with command-line arguments')
+
+    parser.add_argument('--model',
+                        '-m',
+                        type=int,
+                        default=0,
+                        help='Model ID [0-2]')
```

Comment:
we probably want to switch this to allow just passing in the model string. you could use the `choices=models` argument to put the default valid models you've specified below in the arg printout

Reply:
That sounds reasonable.  Mostly this file was for my testing and I wanted it to contain known good models, which are in tight demand and not easy to find, as we've seen.